infrastructure/k8s/helm-chart/eoapi/templates/raster/manifest.yaml

```diff
@@ -68,5 +76,41 @@ spec:
     port: 8080
     targetPort: 8080
   selector:
-    app: raster-{{ .Values.gitSha }}
+    app: raster
+---
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: raster
+  {{- with .Values.raster.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- if .Values.raster.enable_shared_ingress }}
+    alb.ingress.kubernetes.io/group.name: shared-ingress
```

Comment:
comment for reviewer: the toggle for moving over each service ingress to a single ALB